subprojects/junit-jupiter/src/test/java/org/mockitousage/JunitJupiterTest.java

```diff
@@ -70,4 +73,17 @@ void shouldWeCreateMocksInTheParentContext() {
             assertThat(rootMock).isNotNull();
         }
     }
+
+    @Test
+    void should_be_injected_correct_instance_of_mock() {
+        assertThat(classWithDependency.dependency).isEqualTo(rootMock);
```

Comment:
Lets use `isSame` here, as we literally want this object to be there.

Reply:
Good catch, have changed it. Thanks!

Reply:
I have amended the previous commit as it's a simple and small change